core-drawer-panel.html

```diff
@@ -168,14 +168,18 @@
       });
     },
 
+    _mainSelected: function() {
```

Comment:
This doesn't need to live on the prototype. You can wrap this `Polymer` call in an IIFE and declare this function in that scope.